tests/framework/console/FakeController.php

```diff
@@ -84,6 +84,10 @@ public function actionAksi6()
         return $this->testArray;
     }
 
+    public function actionAksi7(self $typedArgument, $simpleArgument) {
```

Comment:
for my approval, this newly added testAction should be altered based on my argument https://github.com/yiisoft/yii2/pull/13772/files#r106026067